src/Endpoints/class-metadata-endpoint.php

```diff
@@ -0,0 +1,70 @@
+<?php
+/**
+ * Metadata endpoint abstract class
+ *
+ * @package Parsely\Endpoints
+ * @since 3.2.0
+ */
+
+declare(strict_types=1);
+
+namespace Parsely\Endpoints;
+
+use Parsely\Parsely;
+
+/**
+ * Metadata endpoint classes are expected to implement the remaining functions of the class.
+ *
+ * @since 3.2.0
+ */
+abstract class Metadata_Endpoint {
+	protected const VERSION    = '1.0.0';
```

Comment:
By putting that here, then consumers will need to check their logic for the endpoints every time we make a breaking change to _any_ endpoint output, even if the endpoint they are calling with the parsely data has not changed.

I would suggest leaving this in the Rest_Metadata for now, since that's really the output we're making claims against - it doesn't have any bearing on changes for GraphQL or other endpoints.

Reply:
Fair point.